templates/addons/calico-dual-stack/values.yaml

```diff
@@ -17,3 +17,10 @@ installation:
       encapsulation: None
       natOutgoing: Enabled
       nodeSelector: all()
+  registry: mcr.microsoft.com/oss
```

Comment:
this seems to work. However, we may need to pin the version as we are now dependent on the image becoming available, which may not always be right when a new release is published

Reply:
It sure would be nice if we could include the chart version here so we could maintain it in one place....

Should we maybe define a Makefile env var that declares the version and then wire that reference in rather than having to update the same string in a bunch of places (and maybe missing some)?

Reply:
I tried but Calico doesn't have the version as part of the values unfortunately :( 

> Should we maybe define a Makefile env var that declares the version and then wire that reference in rather than having to update the same string in a bunch of places (and maybe missing some)?

I agree that would be much better but not sure how feasible that it since all the places are different languages (md for docs, go for e2e test helpers, bash for test scripts, Tiltfile, etc.). I can try.

Reply:
ok was able to reuse the existing Makefile variable everywhere but the docs, PTAL